### DISCUSSION TITLE
[Clue Scroll] Add ability to edit clue hint infobox

### DISCUSSION THREAD

**Discussion by ZeiZeiii** [runelite/runelite#16587] [Ideas]
It would be so nice to allow us edit and personalize clue hint infoboxes.

For example, when there is a step in Legends' Guild, I'd like to have it read "Use Quest Cape", instead of that long, generic helper.

EDIT:
I learned you can set notes. At least it should be highlighted with a custom color. Preferably you could edit the given text, with reset option, thus no need for notes. 

OR

Add checkbox to configure "Show notes only".

**Comment by geheur** [comment]
can't you already shift right click the clue overlay to add a custom hint?

**Comment by ZeiZeiii** [comment]
Tbh did not know this. But why it has to be a note, not an edit with reset option? Note could also be highlighted with custom color maybe?
